app/models/emergency_message.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: emergency_messages
+#
+#  id                    :integer not null, primary key
+#  show                  :boolean not null default: false
+#  message               :text
+#
+
+class EmergencyMessage < ActiveRecord::Base
+
```

Comment:
to my understanding only one emergency message can exists in the db right?

If that's the case, how about to add some check like?
```
before_create :only_one_record_allowed

private
  def only_one_record_allowed
    errors.add(:base, "You can create only one emergency message") if EmergencyMessage.count > 1
  end
```

@zaparka what do you think?